src/Sulu/Bundle/ContentBundle/Controller/NodeController.php

```diff
@@ -604,6 +606,15 @@ public function putAction(Request $request, $uuid)
             return $this->handleView($this->view($e->toArray(), 404));
         }
 
+        if ($request->query->get('force', false) === false
+            && $request->request->has('_hash')
+            && $this->get('sulu_hash.auditable_hasher')->hash($document) !== $request->request->get('_hash')
+        ) {
+            $e = new InvalidHashException(get_class($document), $document->getUuid());
+
+            return $this->handleView($this->view($e->toArray(), 409));
```

Comment:
Would be cool to use the [exception feature of the FOSRestBundle](http://symfony.com/doc/current/bundles/FOSRestBundle/4-exception-controller-support.html) here, but they just set the HTTP Status code in the response, and it it is not too easy to find out how to adapt that. Maybe that would be a general topic for a hackday :smiley: